core/App/types/oca.ts

```diff
@@ -13,15 +13,21 @@ export enum BaseType {
   DATEINT = 'DateInt',
 }
 
-export enum OverlayType {
+export enum BaseOverlayType {
```

Comment:
Mentioned above but I think in the project and TS in general we favour `Bla.Foo` rover `Bla.FOO`.